tests/foreman/ui/test_rhc.py

```diff
@@ -65,10 +66,18 @@ def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
         module_target_sat.cli.Subscription.upload(
             {'file': manifests_path, 'organization-id': module_rhc_org.id}
         )
+        # Enable and sync required repos
+        repo1_id = enable_sync_redhat_repo(REPOS['rhel8_aps'], module_rhc_org.id)
+        repo2_id = enable_sync_redhat_repo(REPOS['rhel7'], module_rhc_org.id)
```

Comment:
Is it necessary to sync these large repos for these tests?

Reply:
yes, it's necessary. :)

Reply:
yes, please

Reply:
it's required for iqe vm's when we will execute their tests that's why I need repos already synced.